docs/UserGuide.md

```diff
@@ -69,59 +72,141 @@ Currently, **TrackO** only supports tracking inventory items that are countable
+at any commands you need!
+
+p.s. Not too sure about some technical terms in this guide? Check out our [glossary](#glossary) to get better understand all the technical jargon! 
+
+-------------------------------------------------------------------------------------------------------------------
 ## Quick start
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `TrackO.jar` from [here](https://github.com/AY2223S1-CS2103T-W15-3/tp/releases).
+2. Download the latest `TrackO.jar` from [here](https://github.com/AY2223S1-CS2103T-W15-3/tp/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your TrackO. { INSERT SCREENSHOTS HERE }
+3. Copy the file to the folder you want to use as the _home folder_ for your TrackO. { INSERT SCREENSHOTS HERE }
 
-1. Double-click the file to start the app. The GUI similar to the one below should appear in a few seconds. Note how the app contains some sample data.<br>
+4. Double-click the file to start the app. The GUI similar to the one below should appear in a few seconds. The application will already contain some sample
+   data.<br>
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`listo`** and pressing Enter will show a list of existing orders.<br>
-   Some example commands you can try:
+5. To start out with some commands, head over to the [tutorial segment](#tutorial) of this user guide!
 
-    * **`listo`** : Lists all orders.
+6. If you'd like to skip the tutorial, head over to the [features section](#features) below for some of the 
+more technical specifications of each command.
 
-    * **`addi`**`n/Keychain q/200 d/This is a metal keychain sp/15.00 cp/10.00` : Adds an inventory item named `Keychain`, which has quantity `200`, description `This is a metal keychain`, selling price `$15.00` and cost price `$10.00` to the inventory list.
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+Want to begin integration with TrackO now? Enter the `clear` command to get rid of the sample data.
+</div>
 
-    * **`deleteo`**`3` : Deletes the 3rd order shown in the current list.
+--------------------------------------------------------------------------------------------------------------------
+## Layout
 
-    * **`exit`** : Exits the app.
+--------------------------------------------------------------------------------------------------------------------
+## Tutorial
+
+To get you started, here are some simple commands you can try out on our sample data before getting to know more of
+TrackO's functionalities! 
+
+1. We start off by getting to know how to manage our **inventory**!
+   1. Adding an item: `addi`
+       * Say your small business has `350` of a new item, `Teddy bear keychain` currently in your physical inventory.
+         It cost you `$0.89` to make these keychains and are selling them at `$3.50`.
```

Comment:
It costs you...... and your selling price for each unit is `$3.50`

Reply:
Perhaps "costed" ? 
Or "costs" you...per unit

Reply:
I think 'costs' is better!